### DISCUSSION TITLE
Windows Search Provider

### DISCUSSION THREAD

**Discussion by RihanArfan** [spacedriveapp/spacedrive#2259] [Feature Request]
It'd be fantastic if Spacedrive provided a Windows Search provider. This allows replacing Windows Search with 100% custom UI, which with Spacedrice could search files and locations across devices, etc.

https://learn.microsoft.com/en-us/windows/apps/develop/search/search-providers

I originally shared this idea on Discord [here](https://discord.com/channels/949090953497567312/949090953950560307/1215289427048276038).

**Comment by utkubakir** [comment]
Great idea! We will definitely do this sometime in the future.
